src/internal/throwError.js

```diff
@@ -0,0 +1,3 @@
+export default message => {
```

Comment:
Not cool with this. I think we talked about it in past. This is a side effect. If we want to introduce the side effect in our functions like `inRange` (which I am for) the side effect (error) should originate in that function and not in some internal `throwError` function. Every stacktract will start at line `2` of `trowError.js`. Been there, done that and falled back to throwing the error from the place where it should be thrown.

Reply:
OK. Though it would be alright internally but I see your argument.